src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -7,7 +7,6 @@
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)'=='netfx'">true</IsPartialFacadeAssembly>
-    <PackageTargetFramework Condition="'$(TargetGroup)' == 'netcoreapp'">netcoreapp;portable-net45+win8+wp8+wpa81</PackageTargetFramework>
```

Comment:
I needed it for an earlier version of the change where I was using GetFilesToPackage.  I noticed that it was using an invalid TFM (netcoreapp with no version).  Moreover the property isn't needed at all because the library doesn't have a pkgproj.